modules/tinymce/src/themes/silver/main/ts/ContextToolbar.ts

```diff
@@ -323,6 +323,13 @@ const register = (editor: Editor, registryContextToolbars, sink: AlloyComponent,
       }
     });
 
+    editor.on('AfterProgressState', (event) => {
+      if (event.state) {
+        lastAnchor.set(Optional.none());
```

Comment:
This is duplicate logic so could we please abstract this out into a single function?

Reply:
Have done, called it `close` because I *think* that's what clearing the anchor (in combination with hiding the element) does. If you happen to have more wisdom around this and have a better idea I'm quite ready to rename it.